index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+  <link rel="stylesheet" href="css/style.css">
+  <body>
+    <nav id="navbar"><h1>  Firts Navbar</h1>
+      <ul>
+        <li><a href="#welcome-section">Welcome</a></li>
+        <li><a href="#projects">Projects</a></li>
+      </ul>
+    </nav>
+    <a href="https://github.com/AntonioHincapie/Muhammad_Marco_CollaborativeWork.git" target="_blank"  id="profile-link"><h4>GitHub</h4></a>
+   
+    <div id="welcome-section">
+      <h1>Hey I am Mimic</h1>
+      <h2>Lorem Imspum!</h2>
+    </div>
+    <section id="projects">
+      <div id="image">
+        <img src="img/landscape.jpg" alt="Beatiful Landscape">
+      </div>
+      <div class="project-tile">
+       <a href="https://codepen.io/freeCodeCamp/full/zNBOYG">Project Link</a> 
+      </div>
+    </section>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
```

Comment:
Use of detailed comments may arrive at beginning of the file or they should be displayed in front of the code to or for a reason at the beginning of a code portion.

Reply:
- Please @for-ashraf try to use polite comments, so as to make your review friendly and not be scary to the student
##### Good
- This is a polite suggestion: *"I think it would be better to use semantic tags"*
##### Bad
- This reads as an order: *"Use semantic tags"